src/ray/protobuf/gcs.proto

```diff
@@ -227,6 +227,12 @@ message JobTableData {
   bytes job_id = 1;
   // Whether it's dead.
   bool is_dead = 2;
+  // The UNIX timestamp corresponding to this event (job added or removed).
+  int64 timestamp = 3;
```

Comment:
It's better to declare 2 fields `start_timestamp` and `stop_timestamp` explicitly instead of 1 `timestamp`.

Reply:
Each entry in the JobTable is a log entry (either job addition, or job removal, if is_dead = true), so a single timestamp is indeed the more natural representation here. They get aggregated into start time and stop time in the client API.